routes/items.js

```diff
@@ -35,4 +35,15 @@ router.post( '/:id/completed', ( request, response ) => {
     .catch( error => response.json({ success: false, id, message: error.message }))
 })
 
+router.post( '/:id/edit', ( request, response ) => {
+  const Item = request.app.get( 'models' ).Item
+
+  const { id } = request.params
+  const { title, description } = request.body
```

Comment:
I mean validate that we are only sending the currently known and allowed parameters to the `Item.update` call - `completed`, `title`, and `description`.  Yes, this would be part of the scope of this issue.  I haven't tested this code, but you could do something like this:
```js
const VALID_PARAMETERS = [ 'completed', 'title', 'description' ]
const updateParameters = Object.entries( request.body ).reduce( (memo, entry) => {
  if( VALID_PARAMETERS.includes( entry[ 0 ] ) ) {
    memo[ entry[ 0 ] ] = entry [ 1 ]
  }

  return memo
})

Item.update( updateParameters, { where: { /* etc. */ } } )
```